src/env_state.F90

```diff
@@ -73,6 +73,8 @@ module pmc_env_state
      real(kind=dp) :: solar_zenith_angle
      !> Box height (m).
      real(kind=dp) :: height
+     !> Scaling coefficient for additive coagulation kernel.
+     real(kind=dp) :: beta_1
```

Comment:
I don't like the name of this variable (it's very mysterious until you read the comment). Could we call it something like `additive_kernel_coefficient` instead?

Reply:
Changed from `beta_1` to `additive_kernel_coefficient` throughout code in https://github.com/compdyn/partmc/pull/187/commits/5177a4646d60a7374f4e27dcc120c4d4da43da30